app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -9,13 +9,16 @@ import NoteTable from "./NoteTable";
 import NewNotePane from "./NewNotePane";
 import DeleteAlert from "./DeleteAlert";
 
+import notesInitialList from "constants/notesInitialList";
+import sortOptions from "constants/sortOptions";
+
```

Comment:
Please sort all import statements in the correct order. Please refer [Sorting import statements](https://www.bigbinary.com/react-best-practices/sort-import-statements#sorting-import-statements) for more details.